### PR TITLE
Wiki doc

### PR DESCRIPTION
**Pull request by maf-embraceit**

### CODE REVIEW

**Review by shahbaz-mehar — COMMENTED** [review]
Changes have been reviewed.

File: wiki.md [hunk 1]
```diff
@@ -0,0 +1,176 @@
...
+
+
+# Prerequisites
+
+Minimum system requirements are:
+- WordPress min. 5.0 – max. 5.8.2
+- WooCommerce min. 3.9.0 - max. 5.9.0
+- PHP 7.0+
+- PHP-bcmath library installed.
+- PHP-curl MUST be enabled.
```
Comment (emicha) [inline wiki.md:33]: In order to be consistent, shouldn't this be part of the `Supported Versions` section like we did in Magento2?

File: wiki.md [hunk 2]
```diff
@@ -0,0 +1,176 @@
...
+
+The latest tested version is:
+- WordPress 5.8.2, WooCommerce 5.9.0 and PHP 7.4
```
Comment (emicha) [inline wiki.md:36]: Same above

File: wiki.md (hunk 1 above)
Comment (maf-embraceit) [reply wiki.md]: added  under Supported versions section

Comment (emicha) [reply wiki.md]: @maf-embraceit is the table of contents updated with it?

Comment (maf-embraceit) [reply wiki.md]: @emicha I've updated the table of contents.

**Review by emicha — APPROVED** [review]

**Pull request merged**